snapcraft/internal/build_providers/images.py

```diff
@@ -0,0 +1,137 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
Can you elaborate a little regarding the placement of this file? We have `build_providers/_qemu` and `build_providers/_multipass`, and now a `build_providers/images.py` that is very qemu-specific? What is the plan for multipass support with this interface?

Reply:
We will use these to launch the mutlipass images too.